workshops/popup_chrome_extension/README.md

```diff
@@ -0,0 +1,498 @@
+              <div id="area" class="col">
+                <p id="none1" class="text-center" style="padding-right: 0px;padding-left: 0px;text-align: center;width: 350px;min-width: 350px;max-width: 350px;">No bookmarks yet! Add some :)</p>
+              </div>
+          </div>
+          </div>
+        </div>
+<br>
+        <p class="text-center" style="color: #7d8285; padding-right: 0px;padding-left: 0px;text-align: center;">made with ❤️️ by <a href="mailto:email@example.com">your mom :)</a></p>
+
+<div id="data">
+</div>
+```
+
+This is pretty similar to the Add Meeting component in terms of what tags are present and how to use them. **Tip: You should add your name and email hyperlink into your code in case people have questions or need to reach out :)**
+
+![Last component](https://cloud-fodkt6aig.vercel.app/0image.png)
+
+Now that you've finished the `index.html` code, we can start to work on the backend. If you need a snip of what the final code I wrote should look like, you can see this link: https://gist.githubusercontent.com/shreyfirst/04ea57ac3bb6db7b0a7db52881256932/raw/41749144bd561a9acca42309960592f4a7dead36/Zoom%2520bookmarks%2520demo
+
+**This is what everything should look like in Repl!**
+
+![Final Repl design of Zoom Bookmarks](https://cloud-3lh6geocf.vercel.app/0image.png)
+
+## Backend
+
+Our backend is inside of `action.js`.
+
+### Structure
+
+Let's plan out what the backend is going to look like:
+
+1. Adding an element to the new bookmarks section
+2. Removing an element from the bookmarks section
+3. Having a popup for copy and pasting
+4. Function to copy onto the user's clipboard
+5. Open a meeting link in a new tab
+6. Waiting for a user to click a button
+
+There are obviously other things we need to do with this app, but this is the main list of items we will be basing off the functions for. The rest will be formatting and fixing around. We will be using `addEventListener`s into the code pretty often. Event Listeners do exactly what they sound like, they listen for events in the code. This include button clicks!
+
+### Writing Functions
+
+The first thing we need to write is an event listener that says: "when the content is loaded" - we don't want our other event listeners later to get confused because the page hasn't loaded yet. This will wrap our enter code
+
+```javascript
+document.addEventListener('DOMContentLoaded', function () {
+
+	// code here
+
+});
+```
+
+We are going to need to create elements many times in this application. Elements are just more tags, but we add them programtically. Here is how to add and remove elements in code:
+
+```javascript
+let elements = [];
+function addElement  (parentId, elementTag, elementId, html) {
+
+  var p = document.getElementById(parentId);
+  var newElement = document.createElement(elementTag);
+  newElement.setAttribute('id', elementId);
+  newElement.innerHTML = html;
+  p.appendChild(newElement);
+
+  elements.push(elementId)
+  if (elements.length==1)
+  {
```

Comment:
Inconsistent opening curly brace. A few lines before it was on the same line, now it's on a new line. Make sure all of the code formatting and stuff is consistent